plugins/catalog/src/components/EntityPage/EntityPage.tsx

```diff
@@ -63,6 +64,16 @@ export const getPageTheme = (entity?: Entity): PageTheme => {
   return pageTheme[themeKey] ?? pageTheme.home;
 };
 
+const EntityPageTitle: FC<{ title: string; entity: Entity | undefined }> = ({
+  entity,
+  title,
+}) => (
+  <Box display="inline-flex" alignItems="center" height="1em">
```

Comment:
does one need the height? should work without that, no?

Reply:
Yes - works without height, but than the header height jumps.

![SM53WIDua7](https://user-images.githubusercontent.com/8904624/86370589-3ccb1780-bc80-11ea-90d1-fa0fb2d064f2.gif)

If this is ok, I will change it. Alternative to this could be a fixed height for the header (`height: 118` instead `minHeight: 118`)

Reply:
Nah, in that case your fix is way more semantically clean.